internal/node/controller/controller.go

```diff
@@ -23,10 +23,10 @@ type NodeController interface {
 	GetUserByUsername(username string) (*node.User, error)
 
 	InstallApp(userID string, newApp *node.AppInstallation, newProxyRules []*node.ReverseProxyRule) error
-	FinishAppInstallation(userID string, appID, registryURLBase, registryPackageID string) error
+	FinishAppInstallation(userID string, appID, registryURLBase, registryPackageID string, startAfterInstall bool) error
 	GetAppByID(appID string) (*node.AppInstallation, error)
 
-	StartProcess(process *node.Process) error
+	StartProcess(appID string) error
```

Comment:
What's the difference between an app and process again? We have some internal processes that aren't apps but all apps are processes?

Reply:
App or `AppInstallation` is how we keep track of what's installed by the user. A process is just an instance of an app running. In the future, we may have processes that aren't tied to apps, but that doesn't exist right now